addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -654,10 +654,19 @@ odoo.define('website.s_website_form', function (require) {
             return () => {
                 // To be visible, at least one field with the dependency name must be visible.
                 const dependencyVisibilityFunction = this._visibilityFunctionByFieldName.get(dependencyName);
-                const dependencyIsVisible = !dependencyVisibilityFunction || dependencyVisibilityFunction();
-                if (!dependencyIsVisible) {
+                const dependencyPassesVisibilityTest = !dependencyVisibilityFunction ||
+                                                        dependencyVisibilityFunction();
+                if (!dependencyPassesVisibilityTest) {
```

Comment:
what is the purpose of this? I don't know if it's appropriate to do this kind of refactoring in stable...